e2e/ansible/roles/kubernetes-centos/tasks/1

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
What is this file? Is it a tmp file or main.yml got renamed?

Reply:
Seems like a junk file. Removed it.